Add first draft of analysis plan (#9)
nipraxis-spring-2022/diagnostics-totally-human: analysis_plan.md

# First analysis plan

> This is the suggested analysis plan foe the outlier detection

Reviewer: ```suggestion
This is the suggested analysis plan for the outlier detection
```

## Reading we need to do

> * Look at Nipy's diagnostics (put link in ther)

Reviewer: ```suggestion
* Look at [Nipy's diagnostics](https://nipy.org/nipy/api/generated/nipy.algorithms.diagnostics.screens.html)
```

> * Reviwe Principal Component Analysis (Where)

Reviewer: ```suggestion
* Review [Principal Component Analysis](https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html)
```

> * Review FSL's ICA method

Reviewer: ```suggestion
* Review [FSL's ICA](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/MELODIC) method
```

> 

Reviewer: ```suggestion
* Check AFNI's [outlier count](https://afni.nimh.nih.gov/pub/dist/doc/program_help/3dToutcount.html)
```

## Action plan

> Use simplest possible outlier detection to get `find_outlier.py`to wprk.

Reviewer: ```suggestion
Use the simplest possible outlier detection to get `find_outlier.py` to work.
```
